systemd/ubuntu-advantage.service

```diff
@@ -25,6 +25,7 @@ ConditionPathExists=!/var/lib/ubuntu-advantage/private/machine-token.json
 # The following conditions correspond to those two modes.
 ConditionPathExists=|/run/cloud-init/cloud-id-gce
 ConditionPathExists=|/run/cloud-init/cloud-id-azure
+ConditionPathExists=|/run/cloud-init/cloud-id-lxd
```

Comment:
The top of the file says "# This service runs on GCP and Azure to enable auto-attaching to Ubuntu Pro". I guess now it can run on lxd instances as well, right?

Reply:
Can someone elaborate on how this daemon will work in the lxd case? The daemon was a concern in older SRUs, but was limited o GCE IIRC. Now I see it in azure as well, and lxd with this change.

Reply:
@panlinux I linked the internal spec in the discussion on MM  please let me know if, after reading that, is there any other case to cover